components/table/table-wrapper.js

```diff
@@ -225,12 +220,31 @@ export class TableWrapper extends RtlMixin(SelectionMixin(LitElement)) {
 				display: none;
 			}
 			:host([type="light"]) {
+				--d2l-table-border-radius: 0rem; /* stylelint-disable-line length-zero-no-unit */
+				--d2l-table-border-radius-sticky-offset: 0rem; /* stylelint-disable-line length-zero-no-unit */
```

Comment:
Did the `calc`s not work with `0`?

Reply:
Haha frustratingly no - `calc()` seems to need units, even for 0.

Reply:
A bit more context on this btw:
https://stackoverflow.com/a/32517653

`calc()` needs all values to be of the same type. 0 is an `integer` type, which can't be added/subtracted with a `length` type (e.g. `6px`).

Reply:
Interesting! Feels like the browsers still could have been like "I know what you're trying to do here, I'll help you out", but no.